test/static/turtle_article_errors.html

```diff
@@ -21,6 +21,8 @@
 <meta name="description" badcontent="Exposition on the awesomeness of turtles"/>
 <meta name="keywords" badcontent="turtle, movie" />
 
+<link rel="apple-touch-icon" badhref="movieturtleapple.png">
+<link rel="icon" badhref="movieturtle.png" badtype="image/png">
```

Comment:
<link rel="apple-touch-icon" href="" sizes="">
<link rel="icon" href="" sizes="" type="">

Might be good to add here too.